packages/runtime/src/useCases/consumption/requests/AcceptIncomingRequest.ts

```diff
@@ -20,6 +25,29 @@ export class AcceptIncomingRequestUseCase extends UseCase<AcceptIncomingRequestR
             return Result.fail(RuntimeErrors.general.recordNotFound(LocalRequest));
         }
 
+        if (
+            localRequest.source?.type === "RelationshipTemplate" &&
+            ![LocalRequestStatus.Decided, LocalRequestStatus.Completed, LocalRequestStatus.Expired].includes(localRequest.status)
```

Comment:
I think @britsta initially tried this, but the problem is that we can't get the RelationshipTemplate and its properties inside of the IncomingRequestsController. But you are right that there the RequestStatus is already checked, so we don't have to do it in the use case again. Then, I'll just remove Decided since we agreed that rejected (decided) Requests should also expire if they are not completed once the RelationshipTemplate expires.

Reply:
I would have guessed that getting the template inside the controller would work just like getting a relationship inside the controller (there's a relationshipResolver property)